core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -175,13 +177,22 @@ class Building : NamedStats(), IConstruction {
             else
                 for (unique in civInfo.getMatchingUniques("[] from every Wonder"))
                     stats.add(unique.stats)
-
+            
+            if (city != null) {
+                for (unique in city.religion.getMatchingUniques("[] from every []")) {
```

Comment:
This too, add to line 162. There's an 'if' in the way so it's a little longer, like:

var fromEveryUniques = civInfo.getMatchingUniques("[] from every []")
if(city != null) fromEveryUniques += city.religion.getMatchingUniques("[] from every []")

for (unique in fromEveryUniques)
   if (matchesFilter(unique.params[1]))
      stats.add(unique.stats)